src/components/join-flow/registration-error-step.jsx

```diff
@@ -14,6 +14,9 @@ class RegistrationErrorStep extends React.Component {
             'handleSubmit'
         ]);
     }
+    componentDidMount () {
+        this.props.sendAnalytics('join-error');
```

Comment:
wrap this with `if (this.props.sendAnalytics) {...}` ?

Reply:
(also applies to most other individual steps)

Reply:
If I make sendAnalytics required like you suggested below, would you still suggest adding the if?  I think you're right that it should be required.